lib/ae_page_objects/core/dsl.rb

```diff
@@ -20,7 +20,9 @@ def element(name, options = {}, &block)
 
       self.element_attributes[name.to_sym] = klass
 
-      define_method name do |&block|
+      define_method name do |&block_for_define_method|
```

Comment:
@dtognazzini  I think the purpose it that if the user of aepos pass a block, the method should throw exception to notify the use that he cannot pass block to it